mora/service/address.py

```diff
@@ -256,3 +262,78 @@ def edit_address(employee_uuid, req):
     }
 
     c.bruger.update(payload, employee_uuid)
+
+
+@blueprint.route('/address_lookup/')
+@util.restrictargs('local', required=['query'])
```

Comment:
Regardless, this definitely shouldn't be called `local` — `org` or something similar is better. I also think we should a simple and customary `q` for the query parameter.